src/balatro_gym/interfaces.py

```diff
@@ -221,7 +227,7 @@ class BoardState(HasReset):
     num_discards: int
     hand_size: int
     vouchers: Sequence[Voucher]
-    poker_hands: Sequence[PokerHand]
+    poker_hands: dict[str, PokerHand]
```

Comment:
The mutability scares me but given how much the board state will change, maybe this is inevitable.